src/main/java/org/junitpioneer/jupiter/params/RangeSourceProvider.java

```diff
@@ -55,6 +55,7 @@
 								.stream(annotations.annotationType().getAnnotationsByType(ArgumentsSource.class))
 								.anyMatch(annotation -> getClass().equals(annotation.value())))
 						.collect(Collectors.toList()))
+				// .filter(Optional::isPresent) <-- Can not accessed in a static context
```

Comment:
Can anyone help me how to implement this `isPresent` check (violation is, that the following `get()` is used without checking if the value is present)

Reply:
haha flatmapping optionals in later java versions is fixing that actually. not sure how to best tackle this

Reply:
The simplest solution would be to just say `.orElse(Collections.emptyList())` and let it throw a `PreconditionViolationException`?

Reply:
I mean, the `map` should have an empty list anyways(?), so (while somewhat redundant) this is basically the same?

Reply:
> .orElse(Collections.emptyList())

You mean to replace the `.get()` with `orElse(Collections.emptyList())` ?